From 90d6bde5b9e15326c0c688b3eb225f8b22207b10 Mon Sep 17 00:00:00 2001
From: "Pham Thanh Tam [FGA.AIS]" <tampt8@fsoft.com.vn>
Date: Mon, 22 Jul 2019 10:41:32 +0700
Subject: [PATCH 241/628] ARM: DTS: RZG1C r8a77470: Add Fine Display Processor
 (FDP1 module) node to device tree

This patch support Renesas FDP1 kernel module probe.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
Signed-off-by: Pham Thanh Tam [FGA.AIS] <tampt8@fsoft.com.vn>
---
 arch/arm/boot/dts/r8a77470.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/r8a77470.dtsi b/arch/arm/boot/dts/r8a77470.dtsi
index ef9e9b2..ec47b62 100644
--- a/arch/arm/boot/dts/r8a77470.dtsi
+++ b/arch/arm/boot/dts/r8a77470.dtsi
@@ -568,6 +568,14 @@
 			renesas,#ch = <0>;
 		};
 
+		fdp0: fdpm@fe940000 {
+			compatible = "renesas,fdp1";
+			reg = <0 0xfe940000 0 0x2400>;
+			interrupts = <0 262 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp1_clks R8A77470_CLK_FDP1_0>;
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
+		};
+
 		vspdu0: vspdu0@fe930000 {
 			compatible = "renesas,vsp2";
 			reg = <0 0xfe930000 0 0x7404>;
-- 
2.7.4

